DndEncounterSimulator/Objects/Creature.py

```diff
@@ -112,3 +124,28 @@ def find_opponent(self, fighters: List[Creature]) -> Union[None, int]:
             if fighter.camp != self.camp:
                 return index
         return None
+
+    def find_best_weapon(self) -> int:
+        """
+        Method to find the best weapon (ie deals statistically the more damage)
+        #TODO: add options and limitations for two-handed, ranged, damage immunity...
+        :return: (int) the index of the best weapon to use

```

Comment:
Same here: leave a line between the description and the TODO + one line between the TODO and the return